tcii18n/template.py

```diff
@@ -0,0 +1,15 @@
+from .translator import Translator
+
+_cached_translators = {}
+
+
+def flask_methods(app, get_translations_file):
+    @app.context_processor
+    def utility_processor():
+        def trans(sentence):
+            filename = get_translations_file()
+            if filename not in _cached_translators:
```

Comment:
can you check that this actually works?
You could verify that it is skipped if you call twice the same filename.
if it does not work, you should check with the key:

```
if filename not in _cached_translators.keys():
    …
```